var/spack/repos/builtin/packages/py-maestrowf/package.py

```diff
@@ -36,15 +38,18 @@ class PyMaestrowf(PythonPackage):
     version('1.1.0', sha256='1bfec546831f2ef577d7823bb50dcd12622644dad0d3d761998eafd0905b6977')
     version('1.0.1', sha256='dd42ffeac1f0492a576c630b37e5d3593273e59664407f2ebf78d49322d37146')
 
-    depends_on('python@2.7:2.8,3.5:',  type=('build', 'run'))
-    depends_on('py-setuptools',        type='build')
-    depends_on('py-pyyaml@4.2b1:',     type=('build', 'run'))
-    depends_on('py-six',               type=('build', 'run'))
-    depends_on('py-enum34',            type=('build', 'run'), when='^python@:3.3')
-    depends_on('py-enum34',            type=('build', 'run'), when='@:1.1.3')
-    depends_on('py-tabulate',          type=('build', 'run'), when='@1.1.0:')
-    depends_on('py-filelock',          type=('build', 'run'), when='@1.1.0:')
-    depends_on('py-coloredlogs',       type=('build', 'run'), when='@1.1.7:')
-    depends_on('py-chainmap',          type=('build', 'run'), when='@1.1.7: ^python@:2')
-    depends_on('py-dill',              type=('build', 'run'), when='@1.1.7:')
-    depends_on('py-jsonschema@3.2.0:', type=('build', 'run'), when='@1.1.7:')
+    depends_on('python@2.7:2.8,3.5:',   type=('build', 'run'))
+    depends_on('python@3.6:',           type=('build', 'run'), when='@1.1.9:')
+    depends_on('py-setuptools',         type=('build'), when='@:1.1.8')
+    depends_on('py-poetry@0.12:', type=('build'), when='@1.1.9:')
+    depends_on('py-cryptography@:2',    type=('build', 'run'))
```

Comment:
I think this is only needed at build-time?

Reply:
Thanks -- I was on the fence so I added `run` to be safe.